bootstrap.ts

```diff
@@ -1,8 +1,8 @@
 global._ = require("lodash");
 global.$injector = require("./yok").injector;
+$injector.require("options", "./options");
```

Comment:
this should not be here, it will not work for appbuilder-cli and nativescript-cli if you keep it this way